enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/HighAvailabilityModeSwitcher.java

```diff
@@ -349,6 +349,7 @@ public void run()
                     ClusterClient clusterClient = dependencyResolver.resolveDependency( ClusterClient.class );
                     try
                     {
+                        // TODO I doubt this actually works
```

Comment:
It shuts down, but there's no restart or recovery, so this server is essentially bricked.